elements/pfe-navigation/demo/pfe-navigation.story.js

```diff
@@ -0,0 +1,210 @@
+import { storiesOf } from "@storybook/polymer";
+import * as storybookBridge from "@storybook/addon-knobs/polymer";
+import * as tools from "../../../.storybook/utils.js";
+
+import PfeNavigation from "../pfe-navigation";
+
+const stories = storiesOf("Navigation", module);
+
+// Define the template to be used
+const template = (data = {}) => {
+  return tools.component(PfeNavigation.tag, data.prop, data.slots);
+};
+
+stories.addDecorator(storybookBridge.withKnobs);
+
+stories.add(PfeNavigation.tag, () => {
+  let config = {};
+  const props = PfeNavigation.properties;
+
+  // Trigger the auto generation of the knobs for attributes
+  config.prop = tools.autoPropKnobs(props, storybookBridge);
+
+  const slots = PfeNavigation.slots;
+
+  let slotCheck = {};
+  Object.keys(slots).forEach(slot => {
+    if(!slot.startsWith("mobile-") && !slot.startsWith("main")) {
+      slotCheck[slot] = storybookBridge.boolean(`${slots[slot].title}`, true);
+    }
+  });
+
+  // Trigger the auto generation of the knobs for slots
+  // config.has = tools.autoContentKnobs(slots, storybookBridge);
+
+  config.slots = [];
+
+  let skip = slotCheck.skip ? tools.customTag({
+    tag: "div",
+    slot: "skip",
+    content: tools.customTag({
+      tag: "a",
+      attributes: {
+        href: "#rh-main-content"
+      },
+      content: "Skip to content"
+    })
+  }) : "";
+
+  let logo = slotCheck.logo ? tools.customTag({
+    tag: "h2",
+    slot: "logo",
+    content: tools.customTag({
+      tag: "a",
+      attributes: {
+        href: "#"
+      },
+      content: `<img class="logo" src="https://via.placeholder.com/150x50.png" class="screen" title="A logo"/>`
+    })
+  }) : "";
+
+  let search = slotCheck.search ? `<pfe-navigation-item slot="search" pfe-icon="search">
+  <h3 slot="trigger"><a href="#url-to-search-page">Search</a></h3>
+  <div slot="tray" hidden>
+    <div class="container">
+      <form>
+        <input type="text" name="search" value="" placeholder="Enter your search term" style="height: 30px; width: 60%; margin-right: 10px;">
+        <pfe-cta priority="primary"><a href="#">Search</a></pfe-cta>
+      </form>
+    </div>
+  </div>
+</pfe-navigation-item>` : "";
+
+  let main = `<nav slot="main" aria-label="Main">
+  <pfe-navigation-main>
+    <ul>
+      <li>
+        <pfe-navigation-item>
+          <h2 slot="trigger"><a href="#">Products</a></h2>
+          <div slot="tray" hidden>
+              <div class="container test-overflow">
+                <div columns="3">
+                  <div stacked>
+                    <pfe-link-list>
```

Comment:
Does this need to be updated since we removed the pfe-link-list custom element?

Reply:
Yes, it does!